api-catalog-ui/frontend/package.json

```diff
@@ -34,27 +36,25 @@
         "redux-persist": "5.10.0",
         "redux-persist-transform-filter": "0.0.18",
         "redux-thunk": "2.3.0",
-        "rimraf": "3.0.2",
         "rxjs": "6.3.3",
         "swagger-ui-react": "3.25.0",
         "uuid": "3.3.2"
     },
     "scripts": {
+        "build": "npm run coverage && npm run react-scripts-build",
+        "react-scripts-build": "react-scripts build",
+        "postbuild": "rimraf build/**/*.map",
```

Comment:
Why do we delete source maps?

Reply:
this is just transition of behavior from before the PR. I am not aware of the reason. I will create an issue about source maps and we can discuss the benefits.